xml/art-obs-beginners-guide.xml

```diff
@@ -1006,30 +1056,56 @@ Patch0:         &gitproject;_main.diff</screen>
     <para>Create the branch from &exampleuser_plain;'s home project:</para>
     <screen>&prompt.user2;<command>osc</command> bco &obshome1; &gitproject;</screen>
     <para>This will create a branched package in &obsa; at <uri>&obsbranch2;/&gitproject;</uri>
-     and will checkout a directory <filename>&obsworkdir2;/branches/home/&obsuser1;/&gitproject;</filename>.</para>
+     and will check out a directory
+     <filename>&obsworkdir2;/branches/home/&obsuser1;/&gitproject;</filename>.
```

Comment:
Ok, after discussing this topic with Stefan, I will use the default (with colons).